Exception thrown by Selenium for Firefox driver

Stack trace:
https://gist.github.com/schottsfired/5b930c940518267021d23e93d971df84

```
jnlp-slave        | 14:14:07.215 WARN - Exception: The path to the driver executable must be set by the webdriver.gecko.driver system property; for more information, see https://github.com/mozilla/geckodriver. The latest version can be downloaded from https://github.com/mozilla/geckodriver/releases
jnlp-slave        | 14:14:07.261 INFO - Executing: [new session: Capabilities [{browserName=firefox, version=, platform=ANY, firefox_profile=UEsDBBQACAgIAMNxMUoAAAAAAAAAA...}]])
jnlp-slave        | 14:14:07.265 INFO - Creating a new session for Capabilities [{browserName=firefox, version=, platform=ANY, firefox_profile=UEsDBBQACAgIAMNxMUoAAAAAAAAAA...}]
jnlp-slave        | 14:14:07.272 WARN - Exception thrown
jnlp-slave        | java.util.concurrent.ExecutionException: org.openqa.selenium.WebDriverException: java.lang.reflect.InvocationTargetException
jnlp-slave        | Build info: version: '3.0.1', revision: '1969d75', time: '2016-10-18 09:48:19 -0700'
jnlp-slave        | System info: host: '836578da0f14', ip: '172.18.0.7', os.name: 'Linux', os.arch: 'amd64', os.version: '4.8.15-moby', java.version: '1.8.0_45-internal'
jnlp-slave        | Driver info: driver.version: unknown
jnlp-slave        | 	at java.util.concurrent.FutureTask.report(FutureTask.java:122)
jnlp-slave        | 	at java.util.concurrent.FutureTask.get(FutureTask.java:192)
jnlp-slave        | 	at org.openqa.selenium.remote.server.DefaultSession.execute(DefaultSession.java:183)
jnlp-slave        | 	at org.openqa.selenium.remote.server.DefaultSession.<init>(DefaultSession.java:119)
jnlp-slave        | 	at org.openqa.selenium.remote.server.DefaultSession.createSession(DefaultSession.java:95)
jnlp-slave        | 	at org.openqa.selenium.remote.server.DefaultDriverSessions.newSession(DefaultDriverSessions.java:131)
jnlp-slave        | 	at org.openqa.selenium.remote.server.handler.NewSession.handle(NewSession.java:59)
jnlp-slave        | 	at org.openqa.selenium.remote.server.handler.NewSession.handle(NewSession.java:36)
jnlp-slave        | 	at org.openqa.selenium.remote.server.rest.ResultConfig.handle(ResultConfig.java:111)
jnlp-slave        | 	at org.openqa.selenium.remote.server.JsonHttpCommandHandler.handleRequest(JsonHttpCommandHandler.java:189)
jnlp-slave        | 	at org.openqa.selenium.remote.server.DriverServlet.handleRequest(DriverServlet.java:222)
jnlp-slave        | 	at org.openqa.selenium.remote.server.DriverServlet.doPost(DriverServlet.java:184)
jnlp-slave        | 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:707)
jnlp-slave        | 	at org.openqa.selenium.remote.server.DriverServlet.service(DriverServlet.java:150)
jnlp-slave        | 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
jnlp-slave        | 	at org.seleniumhq.jetty9.servlet.ServletHolder.handle(ServletHolder.java:808)
jnlp-slave        | 	at org.seleniumhq.jetty9.servlet.ServletHandler.doHandle(ServletHandler.java:587)
jnlp-slave        | 	at org.seleniumhq.jetty9.server.handler.ContextHandler.doHandle(ContextHandler.java:1127)
jnlp-slave        | 	at org.seleniumhq.jetty9.servlet.ServletHandler.doScope(ServletHandler.java:515)
jnlp-slave        | 	at org.seleniumhq.jetty9.server.handler.ContextHandler.doScope(ContextHandler.java:1061)
jnlp-slave        | 	at org.seleniumhq.jetty9.server.handler.ScopedHandler.handle(ScopedHandler.java:141)
jnlp-slave        | 	at org.seleniumhq.jetty9.server.handler.HandlerWrapper.handle(HandlerWrapper.java:97)
jnlp-slave        | 	at org.seleniumhq.jetty9.server.Server.handle(Server.java:499)
jnlp-slave        | 	at org.seleniumhq.jetty9.server.HttpChannel.handle(HttpChannel.java:310)
jnlp-slave        | 	at org.seleniumhq.jetty9.server.HttpConnection.onFillable(HttpConnection.java:257)
jnlp-slave        | 	at org.seleniumhq.jetty9.io.AbstractConnection$2.run(AbstractConnection.java:540)
jnlp-slave        | 	at org.seleniumhq.jetty9.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:635)
jnlp-slave        | 	at org.seleniumhq.jetty9.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:555)
jnlp-slave        | 	at java.lang.Thread.run(Thread.java:745)
jnlp-slave        | Caused by: org.openqa.selenium.WebDriverException: java.lang.reflect.InvocationTargetException
jnlp-slave        | Build info: version: '3.0.1', revision: '1969d75', time: '2016-10-18 09:48:19 -0700'
jnlp-slave        | System info: host: '836578da0f14', ip: '172.18.0.7', os.name: 'Linux', os.arch: 'amd64', os.version: '4.8.15-moby', java.version: '1.8.0_45-internal'
jnlp-slave        | Driver info: driver.version: unknown
jnlp-slave        | 	at org.openqa.selenium.remote.server.FirefoxDriverProvider.callConstructor(FirefoxDriverProvider.java:96)
jnlp-slave        | 	at org.openqa.selenium.remote.server.FirefoxDriverProvider.newInstance(FirefoxDriverProvider.java:64)
jnlp-slave        | 	at org.openqa.selenium.remote.server.DefaultDriverFactory.newInstance(DefaultDriverFactory.java:60)
jnlp-slave        | 	at org.openqa.selenium.remote.server.DefaultSession$BrowserCreator.call(DefaultSession.java:222)
jnlp-slave        | 	at org.openqa.selenium.remote.server.DefaultSession$BrowserCreator.call(DefaultSession.java:209)
jnlp-slave        | 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
jnlp-slave        | 	at org.openqa.selenium.remote.server.DefaultSession$1.run(DefaultSession.java:176)
jnlp-slave        | 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
jnlp-slave        | 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
jnlp-slave        | 	... 1 more
jnlp-slave        | Caused by: java.lang.reflect.InvocationTargetException
jnlp-slave        | 	at sun.reflect.GeneratedConstructorAccessor2.newInstance(Unknown Source)
jnlp-slave        | 	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
jnlp-slave        | 	at java.lang.reflect.Constructor.newInstance(Constructor.java:422)
jnlp-slave        | 	at org.openqa.selenium.remote.server.FirefoxDriverProvider.callConstructor(FirefoxDriverProvider.java:86)
jnlp-slave        | 	... 9 more
jnlp-slave        | Caused by: java.lang.IllegalStateException: The path to the driver executable must be set by the webdriver.gecko.driver system property; for more information, see https://github.com/mozilla/geckodriver. The latest version can be downloaded from https://github.com/mozilla/geckodriver/releases
jnlp-slave        | 	at com.google.common.base.Preconditions.checkState(Preconditions.java:199)
jnlp-slave        | 	at org.openqa.selenium.remote.service.DriverService.findExecutable(DriverService.java:109)
jnlp-slave        | 	at org.openqa.selenium.firefox.GeckoDriverService.access$000(GeckoDriverService.java:37)
jnlp-slave        | 	at org.openqa.selenium.firefox.GeckoDriverService$Builder.findDefaultExecutable(GeckoDriverService.java:95)
jnlp-slave        | 	at org.openqa.selenium.remote.service.DriverService$Builder.build(DriverService.java:296)
jnlp-slave        | 	at org.openqa.selenium.firefox.FirefoxDriver.createCommandExecutor(FirefoxDriver.java:277)
jnlp-slave        | 	at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:247)
jnlp-slave        | 	at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:242)
jnlp-slave        | 	at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:135)
jnlp-slave        | 	... 13 more
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -111,6 +111,10 @@ RUN apt-get update -qqy \
     xvfb firefox \
   && rm -rf /var/lib/apt/lists/*
 
+ARG GECKO_DRIVER_VERSION=v0.13.0
+RUN wget -O - "https://github.com/mozilla/geckodriver/releases/download/$GECKO_DRIVER_VERSION/geckodriver-$GECKO_DRIVER_VERSION-linux64.tar.gz" \
+      | tar -xz -C /usr/bin
+
 #========================
 # Selenium Configuration
 #========================

--- a/README.md
+++ b/README.md
@@ -28,6 +28,7 @@ This Docker image is intended to be used with the [Jenkins CloudBees Docker Cust
 -   Bower: 1.8.0
 -   Cloud Foundry CLI (latest) at `/usr/local/bin/cf`: 6.23.1
 -   Firefox (latest) at `/usr/bin/firefox`: Mozilla Firefox 44.0
+-   Firefox Geckodriver v0.13.0 at `/usr/bin/geckodriver`
 -   gcc (latest): 4.9.2-10ubuntu13
 -   Grunt CLI: 1.2.0
 -   Gulp: 3.9.1

